vertx/src/main/java/com/microsoft/gctoolkit/vertx/VertxDataSourceChannel.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+package com.microsoft.gctoolkit.vertx;
+
+import com.microsoft.gctoolkit.message.Channels;
+import com.microsoft.gctoolkit.message.DataSourceChannel;
+import com.microsoft.gctoolkit.message.DataSourceParser;
+
+public class VertxDataSourceChannel extends VertxChannel implements DataSourceChannel {
```

Comment:
javadoc

Reply:
javadoc will be completed as a separate pass